docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -252,10 +257,13 @@ The simulation concept model consists of five main components:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
   - Location: United States
-  - Cohort type: Prospective closed cohort of individuals 15 years and older
+  - Cohort type: Prospective **closed** cohort of individuals aged 15 years and 
+    older (open cohort if we only examine on age 50+ population)
   - Size of largest starting population: 100,000 simulants
-  - Time span: Jan 1, 2011 to Dec 31, 2025
-  - Time step: 28 days to capture the duration of treatment, and time to response. (Time step may be revised after further model development.)
+  - Time span: Jan 1, 2011 to Dec 31, 2025 (Jan 1, 2011 to Jan 1, 2021 is a 
+    10-year long burn-in period)
+  - Time step: 28 days to capture the duration of treatment, and time to 
+    response. (Time step may be revised after further model development.)
```

Comment:
I think at this point we're sticking with a 28 day time step, so can remove this note of potentially changing.